client/src/components/Atoms/Confirm.tsx

```diff
@@ -15,7 +15,7 @@ export const useConfirm = () => {
   // On close, we set options to null.
   const [options, setOptions] = useState<IConfirmOptions | null>(null)
 
-  const confirm = (newOptions: IConfirmOptions) => {
+  const confirm = (newOptions: IConfirmOptions | null) => {
```

Comment:
why this change?

Reply:
Oops, that was left there from some earlier testing.